core/src/main/scala/com/nvidia/spark/rapids/tool/Platform.scala

```diff
@@ -191,7 +284,12 @@ abstract class Platform(var gpuDevice: Option[GpuDevice],
     "3.4.2" -> "342",
     "3.4.3" -> "343",
     "3.5.0" -> "350",
-    "3.5.1" -> "351"
+    "3.5.1" -> "351",
+    "3.5.2" -> "352",
+    "3.5.3" -> "353",
+    "3.5.4" -> "354",
+    "3.5.5" -> "355",
+    "4.0.0" -> "400"
```

Comment:
Regarding #1678, perhaps we should also fix the shim for DB in this PR as well. This will help avoid a bug sooner while #1676 is still being implemented.